packages/dm-core-plugins/src/form/widgets/TextWidget/TextWidget.tsx

```diff
@@ -13,7 +13,16 @@ const TextWidget = (props: TWidget) => {
   const _onChange = ({
     target: { value },
   }: React.ChangeEvent<HTMLInputElement>) => onChange(value === '' ? '' : value)
-  return <TextField {...props} onChange={_onChange} label={label} />
+  return (
```

Comment:
You'r sure we don't need the rest of the props?

Reply:
ups, seems like I forgot to add inputRef, variant and helperText. Wil fix!

I could not use {...props} since the required namePath attribute on the props type TWidget is not a valid prop to TextField from EDS.